aspnet/signalr/overview/getting-started/tutorial-high-frequency-realtime-with-signalr.md

```diff
@@ -1,222 +1,276 @@
-
-
-## Overview
-
-This tutorial demonstrates how to create an application that shares the state of an object with other browsers in real time. The application we'll create is called MoveShape. The MoveShape page will display an HTML Div element that the user can drag; when the user drags the Div, its new position will be sent to the server, which will then tell all other connected clients to update the shape's position to match.
-
-![The application window](tutorial-high-frequency-realtime-with-signalr/_static/image1.png)
-
-The application created in this tutorial is based on a demo by Damian Edwards. A video containing this demo can be seen [here](https://channel9.msdn.com/Series/Building-Web-Apps-with-ASP-NET-Jump-Start/Building-Web-Apps-with-ASPNET-Jump-Start-08-Real-time-Communication-with-SignalR).
-
-The tutorial will start by demonstrating how to send SignalR messages from each event that fires as the shape is dragged. Each connected client will then update the position of the local version of the shape each time a message is received.
-
-While the application will function using this method, this is not a recommended programming model, since there would be no upper limit to the number of messages getting sent, so the clients and server could get overwhelmed with messages and performance would degrade. The displayed animation on the client would also be disjointed, as the shape would be moved instantly by each method, rather than moving smoothly to each new location. Later sections of the tutorial will demonstrate how to create a timer function that restricts the maximum rate at which messages are sent by either the client or server, and how to move the shape smoothly between locations. The final version of the application created in this tutorial can be downloaded from [Code Gallery](https://code.msdn.microsoft.com/SignalR-20-MoveShape-Demo-6285b83a).
-
-This tutorial contains the following sections:
-
-- [Prerequisites](#prerequisites)
-- [Create the project and add the SignalR and JQuery.UI NuGet package](#createtheproject2013)
-- [Create the base application](#baseapp)
-- [Starting the hub when the application starts](#startup2013)
-- [Add the client loop](#clientloop)
-- [Add the server loop](#serverloop)
-- [Add smooth animation on the client](#animation)
-- [Further Steps](#furthersteps)
-
-<a id="prerequisites"></a>
+In this tutorial, you:
+
+> [!div class="checklist"]
+> * Set up the project
+> * Create the base application
+> * Map to the hub when app starts
+> * Add the client
+> * Run the app
+> * Add the client loop
+> * Add the server loop
+> * Add smooth animation
+
+[!INCLUDE [Consider ASP.NET Core SignalR](~/includes/signalr/signalr-version-disambiguation.md)]
 
 ## Prerequisites
 
-This tutorial requires Visual Studio 2013.
+### Visual Studio 2017
+
+If you don't already have it, [download Visual Studio 2017](https://visualstudio.microsoft.com/downloads/) now.
+
+### ASP.NET and web development workload
```

Comment:
This is an optional feature of VS 2017 so it shouldn't be listed as an independent items. The previous one should say VS 2017 with the **ASP.NET and web development** workload. 

Per the [template](https://review.docs.microsoft.com/en-us/help/contribute/templates/dotnet/tutorial-template-dotnet?branch=master), prereqs should be a bulleted list rather than h3 headings. I think I missed this in the earlier reviews.